nephio-webui/_gen_configmap_nephio-webui-config.yaml

```
@@ -5,18 +5,41 @@ metadata:
   namespace: nephio-webui
 data:
   app-config.nephio.yaml: |
+    app:
+      baseUrl: http://localhost:7007
+
     # Backstage backend configuration
     backend:
       # Note that the baseUrl should be the URL that the browser and other clients
       # should use when communicating with the backend, i.e. it needs to be
       # reachable not just from within the backend host, but from all of your
       # callers. When its value is "http://localhost:7007", it's strictly private
       # and can't be reached by others.
-      baseUrl: http://localhost:7007 # kpt-set: ${backend-base-url}
+      baseUrl: http://localhost:7007
```

Maybe this needs to be exported
```suggestion
      baseUrl: ${BASE_URL:-http://localhost:7007}
```

---

Not sure I understand? This is a generated YAML config file, I think we should take whatever we are given. I am not sure that bash syntax works in these config files, either.

---

I remember to have some [COR issues](https://github.com/nephio-project/nephio-packages/pull/22) using `localhost` as value when I tried to expose the WebUI service. So eventually, we have to have a way to control this value, it could be via `kpt eval` command.

---

Yes, this PR does enable that. This file is generated based upon this other file, which allows you to set that: https://github.com/nephio-project/nephio-packages/pull/31/files#diff-4f30743a327dec885732c9c376b023baeebfd8ca20745460256b6c276c99b05bR30